
 This patch fixes bat.pro.in so that bat will install correctly.

 Apply it to version 2.4.4 with:

 cd <bacula-source>
 patch -p0 <2.4.4-bat-install.patch
 ./configure <your options>
 make
 ...
 make install


Index: src/qt-console/bat.pro.in
===================================================================
--- src/qt-console/bat.pro.in	(revision 8326)
+++ src/qt-console/bat.pro.in	(working copy)
@@ -9,9 +9,9 @@
 #    
 CONFIG += qt debug @QWT@
 
-bins.path = /$(DESTDIR)@sbindir@
+bins.path = $(DESTDIR)@sbindir@
 bins.files = ./bat
-confs.path = /$(DESTDIR)@sysconfdir@
+confs.path = $(DESTDIR)@sysconfdir@
 confs.commands = ./install_conf_file   
 
 TEMPLATE     = app
